SYCL/XPTI/basic_event_collection.cpp

```diff
@@ -0,0 +1,125 @@
+// REQUIRES: xptifw, opencl
+// RUN: %clangxx %s -DXPTI_COLLECTOR -L%xptifw_lib -I%xptifw_includes -shared -fPIC -std=c++17 -o %t_collector.dll -lxptifw
+// RUN: %clangxx -fsycl -fsycl-unnamed-lambda -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: env XPTI_TRACE_ENABLED=1 env XPTI_FRAMEWORK_DISPATCHER=%xptifw_dispatcher env XPTI_SUBSCRIBERS=%t_collector.dll env SYCL_DEVICE_FILTER=opencl %t.out | FileCheck %s 2>&1
```

Comment:
```suggestion
// RUN: env XPTI_TRACE_ENABLE=1 env XPTI_FRAMEWORK_DISPATCHER=%xptifw_dispatcher env XPTI_SUBSCRIBERS=%t_collector.dll env SYCL_DEVICE_FILTER=opencl %t.out | FileCheck %s 2>&1
```

XPTI_TRACE_ENABLE=1 ?